lib/src/header_parser/sub_parsers/functiondecl_parser.dart

```diff
@@ -70,8 +70,8 @@ List<Func>? parseFunctionDeclaration(clang_types.CXCursor cursor) {
 
     // Initialized with a single value with no prefix and empty var args.
     var varArgFunctions = [VarArgFunction('', [])];
-    if (clang.clang_isFunctionTypeVariadic(cursor.type()) == 1) {
-      if (config.varArgFunctions.containsKey(funcName)) {
+    if (config.varArgFunctions.containsKey(funcName)) {
+      if (clang.clang_isFunctionTypeVariadic(cursor.type()) == 1) {
```

Comment:
Just discovered this by accident when parsing -
```C
extern void abcd();
```
```txt
[WARNING]: Skipping variadic-argument config for function abcd since its not variadic.
```

This didn't affect the flow, but generated a misleading log for every variadic function there would be in the source code